docs/src/_reference/command-line-interface.md

```diff
@@ -604,6 +604,12 @@ meltano invoke --print-var <PLUGIN_ENVIRONMENT_VARIABLE_1> <PLUGIN_NAME>
 meltano invoke --print-var <PLUGIN_ENVIRONMENT_VARIABLE_1> --print-var <PLUGIN_ENVIRONMENT_VARIABLE_2> <PLUGIN_NAME>
 ```
 
+## `lock`
+
+`meltano lock` creates lock files for all [non-custom](/concepts/plugins#custom-plugins) plugins in the project.
+
+To update your lock artifacts with the latest plugin definitions in Meltano Hub, use the `--update` option.
```

Comment:
```suggestion
To update your lock artifacts with the latest plugin definitions in MeltanoHub, use the `--update` option.
```

In future iterations we'll want to figure out how to better version this, but I like this as a small first iteration!